test/Microsoft.ML.CodeAnalyzer.Tests/Code/NameTest.cs

```diff
@@ -53,35 +54,39 @@ class TypeName
-                diagG.CreateDiagnosticResult(25, 30, "_badEv"),
-                diagG.CreateDiagnosticResult(27, 29, "one"),
-                diagG.CreateDiagnosticResult(27, 39, "three"),
-                diagG.CreateDiagnosticResult(28, 22, "enumb"),
-                diagG.CreateDiagnosticResult(28, 35, "Two_Two"),
-                diagG.CreateDiagnosticResult(28, 44, "_three"),
-                diagG.CreateDiagnosticResult(30, 25, "_m2"),
-                diagG.CreateDiagnosticResult(37, 12, "marco"),
-                diagG.CreateDiagnosticResult(37, 31, "polo"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(7, 11).WithArguments("CLASS"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(9, 20).WithArguments("alice"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(10, 21).WithArguments("_bob"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(11, 22).WithArguments("_chaz"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(12, 30).WithArguments("emily"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(12, 37).WithArguments("_francis"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(17, 21).WithArguments("this_is_not_python"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(18, 21).WithArguments("thisIsNotJava"),
+                VerifyCS.Diagnostic(NameAnalyzer.PrivateFieldName.Rule).WithLocation(22, 30).WithArguments("BadEvent"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(23, 29).WithArguments("bad_event"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(26, 30).WithArguments("_badEv"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(28, 29).WithArguments("one"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(28, 39).WithArguments("three"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(29, 22).WithArguments("enumb"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(29, 35).WithArguments("Two_Two"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(29, 44).WithArguments("_three"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(31, 25).WithArguments("_m2"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(38, 12).WithArguments("marco"),
+                VerifyCS.Diagnostic(NameAnalyzer.GeneralName.Rule).WithLocation(38, 31).WithArguments("polo"),
+            };
+
+            var test = new VerifyCS.Test
+            {
+                TestCode = TestSource,
+                FixedCode = FixedTestSource,
+                NumberOfFixAllIterations = 2,
```

Comment:
📝 This means the application of a Fix All did not actually fix all instances at once. The expected `FixedTestSource` was reached, but it took two Fix All iterations to reach. The test harness forces you to call out these cases because they indicate places where a code fix works but could have improved usability.